packages/actions/resources/lang/en/import.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+return [
+
+    'label' => 'Import :label',
+
+    'modal' => [
+
+        'heading' => 'Import :label',
+
+        'form' => [
+
+            'file' => [
+                'label' => 'File',
+                'placeholder' => 'Upload a .csv file',
+            ],
+
+            'columns' => [
+                'label' => 'Columns',
+                'placeholder' => 'Select a column',
```

Comment:
Shouldn't the placeholder also be plural?

Reply:
No, the label is used on the fieldset that wraps the columns, whereas the placeholder is used on each column map individually

Reply:
Ah right, thanks for clarifying.